### ISSUE TITLE
Accessibility: add label under icons on mobile navbar

### ISSUE THREAD

**Issue by Dehelssey** [BDX-town/elk#40]
Accessibility best practices states that navbar items should have a label.

We should at least offer an option to display them. Or, better, an option to hide them but display them by default.

Like #39 i'll submit this to Elk's main repo as well :)

**Comment by Dehelssey** [comment]
OK, there's already a MR on upstream repo: https://github.com/elk-zone/elk/pull/3120

**Comment by Cl0v1s** [comment]
Yup, it's more an upstream job
